utils/const.ts

```diff
@@ -0,0 +1,32 @@
+export const MONGO_URL =
+  'mongodb+srv://' +
+  process.env.MONGO_USER +
+  ':' +
+  process.env.MONGO_PASSW +
+  '@cluster0-bnobp.mongodb.net/test?retryWrites=true&w=majority'
```

Comment:
Maybe add the mongo hostname (`cluster0-bnobp.mongodb.net`) to an environment variable? `process.env.MONGO_HOST`

Reply:
We can add it to Git secrets, to then change it